lib/rack/request-id-passthrough.rb

```diff
@@ -10,14 +10,14 @@
 # See the License for the specific language governing permissions and limitations under the License.
 require 'securerandom'
 require 'net/http'
-
+require_relative '../../lib/rack-request-id-passthrough/rack-request-id-passthrough'
```

Comment:
Why `../../` to go immediately back into lib? `../rack-request-id-passthrough` should do it, and be easier to parse.